packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -632,3 +632,185 @@ describe('masternode', () => {
     })
   })
 })
+
+describe('sendMany', () => {
+  // NOTE(sp): defid side(c++) does not have much tests for sendmany RPC atm.
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  // util functions NOTE(sp) : may be move to a common util file if possible ??
+  /**
+   * Returns matching utxos for given transaction id and address.
+   *
+   * @param {string} txId transaction id
+   * @param {string} address address
```

Comment:
1 address is enough

Reply:
don't really need JSDoc for testing.

Reply:
@jingyi2811 didn't quite understand your comment. input params for the function are `txId` and `address`

Reply:
It's ok. You may remove the JSDoc.